### PR TITLE
[CNXC-319] Re-design Create Analysis and Dashboard

### PR DESCRIPTION
**Pull request by mslwang**
**Problem:**
We want the create analysis functionality to only occur under the CreateAnalysis page, yet right now we currently still allow it to occur on the Dashboard.

**Solution:**
1. We remove the patient lookup in the dashboard.
2. We notify users that patient lookup can only occur in the CreateAnalysis page and redirect them there through a button.
3. We polish up patient lookup so that it's better to use.

### CODE REVIEW

**Comment by mslwang** [comment]
I rebased to change one of the commit messages but I forgot to squash the merge from pulling from the remote branch, whoops.

File: src/pages/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
   return (
     <Wrapper>
       <PageSection className="page-body">
-        {/* <PageSection className="section-area" variant={PageSectionVariants.light}>
-        </PageSection> */}
+        <PageSection className="section-area" variant={PageSectionVariants.light}>
+          <Alert isInline variant="warning" title="Patient lookup designated to Create Analysis page">
+            <p>To look up a patient and create a new analysis, please go to the Create Analysis page.</p>
+          </Alert>
+          <Button variant="link" isLarge className="pf-u-pl-0"
```
Comment (mslwang) [inline src/pages/Dashboard/Dashboard.tsx:21]: Would this look better if I didn't remove left padding? 
(no left padding)
image
vs
(default left padding)
image

File: src/pages/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
 
+  const history = useHistory();
+
   return (
     <Wrapper>
       <PageSection className="page-body">
-        {/* <PageSection className="section-area" variant={PageSectionVariants.light}>
-        </PageSection> */}
+        <PageSection className="section-area" variant={PageSectionVariants.light}>
+          <Alert isInline variant="warning" title="Patient lookup designated to Create Analysis page">
```
Comment (mslwang) [inline src/pages/Dashboard/Dashboard.tsx:18]: I just used the default Alert component properties for this notification box, but this whole container is pretty wide.
image

**Comment by mslwang** [comment]
Commit 06b1cae ties into CNXC-342 and I think given that at this point in time, it is in fact a wrapper for a component we aren't using anymore, I think it would be best to remove it.

File: src/pages/CreateAnalysisPage/CreateAnalysisPage.tsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ const CreateAnalysisPage = () => {
               </PageSection>
               :
               <PageSection className="page-section-no-results" variant={PageSectionVariants.light}>
-                <Error>{ hasSearched ? "No studies found" : "Search for a patient by entering their MRN above" } </Error>
+                <Error>{ hasSearched ? "No studies found for MRN #" + patientID : "Search for a patient by entering their MRN above" } </Error>
```
Comment (mslwang) [inline src/pages/CreateAnalysisPage/CreateAnalysisPage.tsx:26]: Should add .toUpperCase() to patientID here given CNXC-338 (except for the search bar, every other instance of the MRN should probably be all caps).

Comment (micklynch) [reply src/pages/CreateAnalysisPage/CreateAnalysisPage.tsx:26]: let's just leave this, no need to do it.

**Review by micklynch — APPROVED** [review]

**Pull request merged**